src/main/resources/application-local.yml

```
@@ -54,6 +54,66 @@ spring:
+    temp-file-cleanup-hours: 2              # 임시 파일 정리 주기(시간)
+
+# 캐싱 설정
+cache:
+  caffeine:
+    # 아카이브 데이터 캐시 설정
+    archive:
+      maximum-weight: 250000000             # 250MB 메모리 제한
+      expire-after-write: 2h                # 쓰기 후 만료 시간
+      expire-after-access: 30m              # 접근 후 만료 시간
+      record-stats: true                    # 통계 기록 활성화
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)